andrewinsoul/Fast-Food-ClientSide
# #162028730 configure environment for client side with react

#### What does this PR do?
This PR configures the development environment for client side with React

#### Description of Tasks Completed
- Install Dependencies
- Configure Babel through a .babelrc file
- Configure webpack through a webpack.config.js
- Create the root html file
- Create an entry file (app.js)

#### How Should this be Manually Tested?
```
- Clone Repo
- Run npm install to install all dependencies
- Run npm start
- Your Browser should open with the favicon showing and a simple text: "This is Fast-Food-Fast Homepage" 
```
#### Background Context Provided
- N/A

#### What are the Relevant Pivotal Tracker Stories?
[#162028730](https://www.pivotaltracker.com/n/projects/2226230)
